test/url.js

```diff
@@ -63,4 +63,32 @@ describe('url', function() {
       });
     });
   });
+
+  describe('strip protocol', function () {
```

Comment:
For the tests, instead of using javascript template strings, pass the property on the context to the `fn()` calls:

```js
var data = {testUrl: 'http://foo.bar'};
var expectedResult = '//foo.bar/';
var fn = hbs.compile('{{stripProtocol testUrl}}');
fn(data).should.eql(expectedResult);
```